collectibles-rsk-deployment/utils/avoidMainet.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef */
+const mainnets = [1, 30];
+
+/**
+ * Prevents from running on a mainnet
+ */
+async function avoidMainet() {
+  const { chainId } = await ethers.provider.getNetwork();
+  // both 1 and '1' must match
+  // eslint-disable-next-line eqeqeq
+  if (mainnets.some((id) => id == chainId))
```

Comment:
Instead of using "loose equals" (`==`), consider converting both sides to string, then using strict equals (`===`). This will avoid the need to disable the ESlint rule.